### PR TITLE
Proper parenthetics

### PR DESCRIPTION
**Pull request by han8909227**

### CODE REVIEW

File: properParen/parenthetics.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    def __len__(self):
+        """Return the size of the stack."""
+        return self.count
+
+
+def paren(s):
+    """Check for valid paren combos."""
+    l = Stack()
+    for p in s:
```
Comment (nhuntwalker) [inline properParen/parenthetics.py:49]: Write better variable names. Single-letter variable names aren't acceptable.

File: properParen/parenthetics.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            count += 1
+        elif x == '(':
+            if count == 0:
+                return 1
+            else:
+                count -= 1
+
+    if count == 0:
+        return 0
+    else:
```
Comment (nhuntwalker) [inline properParen/parenthetics.py:64]: This `else` isn't necessary. In fact these four lines could just be
```python
return 0 if count == 0 else -1
```

File: properParen/parenthetics.py (hunk 1 above)
Comment (han8909227) [reply properParen/parenthetics.py:49]: fixed with better names

File: properParen/parenthetics.py (hunk 2 above)
Comment (han8909227) [reply properParen/parenthetics.py:64]: replaced with that codez

**Pull request merged**